kubernetes/skaffold/src/main/resources/schemas/v1alpha1.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
How do we maintain these files in the future? Where are they copied from?

Reply:
they are currently created in the skaffold repo (on each release). This PR is the first pass at quickly getting the plumbing in place and getting support for schema editing. As is, we'd need to copy in new schemas into this repo to get support for new Skaffold versions.

The goal next is to dynamically load them (ideally from a GCS bucket) so that we don't have to maintain anything.

Reply:
Yeah, that'd be better, kind of maintenance burden to do this here in the code, huge set of data checked in the code base.

Reply:
agreed. are you ok with phasing it though (as in deferring the enhancement to another PR)? They haven't automated pushing these to a GCS bucket yet.

Reply:
Sure, let's keep it here now - long waited schema editing is welcome! Good step for people to easier integrate existing projects with the plugin Skaffold features.